src/Adapters/Storage/Postgres/PostgresStorageAdapter.js

```diff
@@ -1366,6 +1370,140 @@ export class PostgresStorageAdapter {
     });
   }
 
+  distinct(className, schema, query, fieldName) {
+    debug('distinct', className, query);
+    let field = fieldName;
+    let column = fieldName;
+    if (fieldName.indexOf('.') >= 0) {
```

Comment:
Although this will work the value won't ever be 0, just -1 or > 1. Doesn't means this needs to be changed, but it could also be `>= 1` for be more specific about what's expected.

Reply:
```
const str = '.';
str.indexOf('.') will return 0; 
```
It will never happen but it covers all cases

Reply:
Alright.